charts/timed/ci/default-values.yaml

```diff
@@ -0,0 +1,8 @@
+frontend:
+  image:
+    tag: v1.0-rc1
```

Comment:
This will only work in CI. I think you should do this change in the main `values.yaml` so we don't need to add a value override in the deploy.